ansible/roles/qe_sap_storage/tasks/generic_tasks/prepare_storage.yml

```diff
@@ -3,12 +3,42 @@
 - debug:
     msg: "{{ item }} "
 
+- name: Determine the root device
+  command: "findmnt -n -o SOURCE /"
+  register: root_device
+  changed_when: false
+
+# Cut any partition suffixes like p1, p2
+- name: Normalize root device name
+  set_fact:
```

Comment:
Please always use full name `ansible.builtin.set_fact`. This is detected as error by Ansible linter that  I'd like to enable